src/server.c

```diff
@@ -5519,8 +5519,12 @@ void closeListeningSockets(int unlink_unix_socket) {
     if (server.cluster_enabled)
         for (j = 0; j < server.cfd.count; j++) close(server.cfd.fd[j]);
     if (unlink_unix_socket && server.unixsocket) {
-        serverLog(LL_NOTICE,"Removing the unix socket file.");
-        unlink(server.unixsocket); /* don't care if this fails */
+        if(unlink(server.unixsocket) == 0) {
+            serverLog(LL_NOTICE,"Removing the unix socket file.");
+        }
+        else {
+            serverLog(LL_WARNING,"Error removing the unix socket file: %s",strerror(errno));
+        }
```

Comment:
Lets leave the original log (notice level) indicating we're attempting to remove the file. Also lets avoid the extra `{}` to stay similar to the existing code and reduce blame noise.
```suggestion
        serverLog(LL_NOTICE,"Removing the unix socket file.");
        if (unlink(server.unixsocket) != 0)
            serverLog(LL_WARNING, "Error removing the unix socket file: %s", strerror(errno));
```

Reply:
Fixed in the last commit